docs/QUICKSTART.md

```diff
@@ -135,76 +158,122 @@ This tutorial demonstrates the functionality of Eraser and validates that non-ru
-
-7. Tear down your cluster
-
-    ```bash
-    kind delete cluster
-    ```
\ No newline at end of file
+## Automatically cleaning images
+
+After deploying Eraser, it will automatically clean images in a regular interval. This interval can be set by `--repeat-period` argument to `eraser-controller-manager`. The default interval is 24 hours.
+
+Eraser will schedule collector pods to each node in the cluster and these pods will collect non-running images in the cluster. Once the collectors are done, results are de-duplicated and stored in `imagecollector-shared` object in `ImageCollector` CRD.
+
+```bash
+kubectl get pods -n eraser-system
+NAME                                         READY   STATUS    RESTARTS   AGE
+collector-kind-control-plane-lv982           1/1     Running   0          14s
+collector-kind-control-plane-ab840           1/1     Running   0          14s
+collector-kind-control-plane-sg352           1/1     Running   0          14s
+eraser-controller-manager-649c756544-bgfds   1/1     Running   0          26s
+```
+
+After collector pods are finished, scanner pod will be scheduled to each node in the cluster.
+
+> If you want to remove all the images periodically, you can skip this step by removing the `--scanner-image` argument.
+
+```bash
+kubectl get pods -n eraser-system
+NAME                                         READY   STATUS    RESTARTS   AGE
+eraser-controller-manager-649c756544-bgfds   1/1     Running   0          36s
+eraser-scanner-78p49-vxb4j                   1/1     Running   0          5s
+```
+
+After scanner pods are finished, Eraser will remove the non-running images from the cluster.
+
+```bash
+kubectl get pods -n eraser-system
+NAME                                         READY   STATUS      RESTARTS   AGE
+eraser-controller-manager-649c756544-bgfds   1/1     Running     0          56s
+eraser-kind-control-plane-lswqn              1/1     Running     0          12s
+eraser-kind-worker-wfqc                      0/1     Running     0          12s
+eraser-kind-worker2-gwbit                    0/1     Running     0          12s
+eraser-scanner-78p49-vxb4j                   0/1     Completed   0          25s
+```
+
+Eraser pods will run to completion and the non-running images will be removed.
+
+```bash
+kubectl get pods -n eraser-system
+NAME                                         READY   STATUS      RESTARTS   AGE
+eraser-controller-manager-649c756544-bgfds   1/1     Running     0          61s
+eraser-kind-control-plane-lswqn              0/1     Completed   0          17s
+eraser-kind-worker-wfqc                      0/1     Completed   0          17s
+eraser-kind-worker2-gwbit                    0/1     Completed   0          17s
+eraser-scanner-78p49-vxb4j                   0/1     Completed   0          30s
+```
+
+## Manually cleaning images
+
+Create an `imagelist` and specify the images you would like to remove. In this case, the image `docker.io/library/hello-world:latest` will be removed.
+
```

Comment:
Should we add that you have to re-deploy eraser with no collector image to manually clean?

Reply:
Isn't it possible to manually clean with collector by just deploying an imagelist?

Reply:
yup that actually wont be a problem now with the locks